core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -198,6 +198,47 @@ public static long parseSpaceSize(String spaceSize) {
   }
 
   /**
+   * Parses a String size to Milliseconds.
+   *
+   * @param timeSize the size of a time, e.g. 1M, 5H, 10D
+   * @return the time size in milliseconds
+   */
+  public static long parseTimeSize(String timeSize) {
+    double alpha = 0.0001;
+    String ori = timeSize;
+    String end = "";
+    int index = timeSize.length() - 1;
```

Comment:
this is inefficient. better way is to use regular expression (see [link](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)). 
what you want, is to create a regexp `"\d+[a-zA-Z]+"`, the first matched group is your `timeSize` and the second matched group is your `end`